source/modules/lesson00/grading.rst

```diff
@@ -3,11 +3,11 @@ Grading
 
 The purpose of grading is to help students and instructors assess a student's
 progress on the class, and to demonstrate that a student is proficient and
-achieved the class objectives.
+has achieved the class objectives.
 
 There are three grades used to rate every assignment:
```

Comment:
We ended up developing a "rubric" for course 1 (primarily the online self-paced version)  -- we may want to see if we can be consistent between the classes.

I can't seem to find it anywhere -- you might ask Debalina or Natasha.

Bu tin any case, we gave credit for the code working, but also some for style and "pythonicity" -- I think it's really important to review and (maybe) score on good structure and style s well as simple functionality.

Reply:
I'll add a TODO to this file to discuss the rubric with Andy.